### DISCUSSION TITLE
[2021/10/12] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#170] [General]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/issues/165 

 _Created from https://github.com/wantedly/frontend-night/issues/65 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by hayaokimura** [comment]
## jser info
https://jser.info/2021/10/12/npm-8.0.0node.js-16-jquery-mobilejquery-ui-eslint-8/

**Reply by izumin5210** [reply]
関連 [jQuery UIとjQuery Mobileがついに開発終了、今後はメンテナンスのみに。jQuery本体は引き続き積極的に開発 － Publickey](https://www.publickey1.jp/blog/21/jquery_uijquery_mobilejquery.html)

**Reply by hayaokimura** [reply]
https://openjsf.org/projects/

思ったよりいろいろなプロジェクトが属してた

**Comment by izumin5210** [comment]
https://github.com/mswjs/msw

- ブラウザ・Node 両方でつかえるモックサーバ
- ブラウザの場合は serviceworker で捕まえる
	- devtool で見れる
- Node では http や xhr を拡張してる？
- アプリケーションはモックのことを知らずにテストできる
	- リクエストするライブラリに依存しない
	- apollo client もリアルな実装のままでテストできる

**Reply by izumin5210** [reply]
最近また service worker 悪用するのが流行ってる
[今Partytownがヤバい。JavaScript Sandboxの未来はどっちだ？](https://zenn.dev/stomita/articles/2c16a53223f3c9)
